ui_framework/doc_site/src/views/button/button_example.jsx

```diff
@@ -3,6 +3,8 @@ import React, {
   PropTypes,
 } from 'react';
 
+import { renderToHtml } from '../../services';
```

Comment:
Good call, I plan on doing that globally to the existing `jsx` files after this PR is merged.